forms-flow-theme/scss/_forms.scss

```diff
@@ -223,4 +223,91 @@ select option:hover {
   font-size: var(--font-size-xs);
   font-weight: var(--font-weight-sm);
   line-height: var(--text-line-height);
+
 }
+//FormInput styles
+
+.form-input-box {
+  display: flex;
+  width: 100%;
+  max-width: 40rem;
+  flex-direction: column;
+  justify-content: center;
+  .form-control-input{
+    display: flex;
+    padding: var(--spacer-050) var(--spacer-100);
+    justify-content: center;
+    align-items: center;
+    align-self: stretch;
+  }
+
+  .form-control {
+    &.is-invalid {
+      background-image: none !important;
+    }
+    &:disabled {
+      border-color: var(--ff-gray-400);
+      background-color: var(--ff-gray-000);
+    }
+  }
+  
+  input{ 
+    border-radius: var(--radius-0100)!important; 
+    border: 1px solid var(--ff-primary, #253DF4);
+
+    &:focus {
+    border:2px solid var(--ff-primary, #253DF4) ;
+    }
+
+    &::placeholder{
+      color: var(--ff-gray-400, #949B9E); 
+      font-family: var(--primary-font);
+      font-size: var(--font-size-sm);
```

Comment:
Remove 
font-family:var(--primary-font) ; //Roboto
font-style: normal;
everywhere. it is added in body

Reply:
updated.